### ISSUE TITLE
should warn if List is not imported

### ISSUE THREAD

**Issue by tharvik** [python/mypy#1427]
```python
 a = [1]  # type: List[int]
```
do not give any warning, but
```python
 a = (1,)  # type: Tuple[int]
```
gives
```
list.py:1: error: Name 'Tuple' is not defined
```

**Comment by kirbyfan64** [comment]
Does the latter work if you do `a = (1,)`?

**Comment by tharvik** [comment]
@kirbyfan64 ho, typo from my part, that's true, I'm editing my post; but the warning is still there.

**Comment by gvanrossum** [comment]
This is a duplicate of bug #999. For your program to be correct, both List and Tuple ought to be imported from the typing module.
